src/tree/render.js

```diff
@@ -231,7 +233,7 @@ const renderTree = async ({ element, layout, app, model }) => {
                 ? d.parent.xActual +
                   nodeSize.width / 2 +
                   siblingSpacing / 2 +
-                  (d.data.childNumber - d.parent.children.length / 2) * (nodeSize.width + siblingSpacing)
+                  (d.data.childNumber - (d.parent.children.length < childrenToShow ? d.parent.children.length : childrenToShow) / 2) * (nodeSize.width + siblingSpacing)
```

Comment:
I see now that we probably should break this function out and and send xActual or yActual in as arguments. So we only have to make changes to one function in the future.